### PR TITLE
Add Vector:Length() checks for applyForce and applyOffsetForce functions

### PR DESCRIPTION
**Pull request by CaptainPRICE**
Temporary fix to prevent servers from crashing (#1189).

### CODE REVIEW

**Comment by CaptainPRICE** [comment]
Test code:
```
interval(1)
#ifdef entity:propFreeze(number)
entity():propFreeze(0)
#else
# Just manually unfreeze E2 after spawned.
#endif
applyForce(vec(1e54))  #entity():applyForce(vec(1e54))
```

File: lua/entities/gmod_wire_expression2/core/entity.lua [hunk 1]
```diff
@@ -459,9 +459,12 @@ e2function void entity:applyForce(vector force)
 	if not validPhysics(this) then return nil end
 	if not isOwner(self, this) then return nil end
 
-	if check( force ) then
+	if check(force) then
 		local phys = this:GetPhysicsObject()
-		phys:ApplyForceCenter(Vector(force[1],force[2],force[3]))
+        force = Vector(force[1], force[2], force[3])
+        if force:Length() ~= math.huge then
+		    phys:ApplyForceCenter(force)
```
Comment (AbigailBuccaneer) [inline lua/entities/gmod_wire_expression2/core/entity.lua]: This PR mixes spaces and tabs

**Comment by AbigailBuccaneer** [comment]
Has anybody submitted a Garrybug about this yet? If so, the code should probably have a comment referencing it, as a sort of "we can revert this commit later" message. Other than that, looks good to me :+1:

**Comment by CaptainPRICE** [comment]
See this @Divran.
From what I see, not all functions checks for `math.huge`.. Only 2 out of 6 (excluding angular force functions).

**Comment by Divran** [comment]
@CaptainPRICE Which ones don't? I just checked and they all do. There's only 4 functions by the way, not 6. applyForce, applyOffsetForce, applyAngForce and applyTorque

**Comment by Divran** [comment]
Closing this because a different fix was used.

**Pull request closed without merging**